package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@redocly/openapi-cli",
-  "version": "0.8.4",
+  "version": "0.8.5",
```

Comment:
we need minor bump to `0.9.0` not patch. But let's add a separate commit for that.

Reply:
ok